tests/unit/components/filters/Datetimefilter.spec.ts

```diff
@@ -43,4 +45,53 @@ describe('Datetimefilter.vue', () => {
     expect(vm.dateRange.startDate).toBe(null)
     expect(vm.dateRange.endDate).toBe(null)
   })
+
+  it('creates a date from a given string', () => {
+    const value = '1/1/2000'
+    // @ts-ignore
+    const date = vm.createDateFromValue(value)
+    expect(date).toEqual(new Date(Date.parse(value)))
+  })
+
+  it('creates a date from a given amount of milliseconds from epoch', () => {
+    const epochValue = 1588068623458
+    // @ts-ignore
+    const date = vm.createDateFromValue(epochValue)
+    expect(date).toEqual(new Date(epochValue))
+  })
+
+  it('creates a representable date string without the time component, if time is set to false', () => {
+    const noTimeProps = {
+      name: 'datetime',
+      label: 'Datetime',
+      collapsed: false,
+      time: false,
+      type: 'date-time-filter'
+    }
+    wrapper = mount(DateTimeFilter, { localVue, propsData: noTimeProps })
+
+    const dateToStringify = new Date(1588068623458)
+    const comparisonDate = dateToStringify.toLocaleDateString()
+    // @ts-ignore
+    const dateString = vm.dateString(dateToStringify)
+    expect(dateString).toEqual(comparisonDate)
+  })
```

Comment:
Ik zou hier misschien eerder `beatifyDateTime` testen in plaats van die dateString functie (zie comment hieronder)

Reply:
yes, too many microfunctions.